README.rst

```diff
@@ -380,7 +382,8 @@ Enabling Centralised Logging
 
 In Kolla-Ansible, centralised logging is easily enabled and results in the
 deployment of Elasticsearch and Kibana services and configuration to forward
-all OpenStack service logging.
+all OpenStack service logging. **Be cautious as Elastic search will consume
+significant portion of available resources on standard deployment.**
```

Comment:
Nit: **a** significant portion of available resources on **a** standard deployment.